pages/blog/swr-v2.en-US.mdx

```diff
@@ -0,0 +1,278 @@
+---
+image:
+description: 'We are excited to introduce SWR v2.0, including features to improve the use experience of your application.'
+date:
+---
+
+import Callout from 'nextra-theme-docs/callout'
+import Bleed from 'nextra-theme-docs/bleed'
+
+import Authors, { Author } from 'components/authors'
+import Video from 'components/video'
+
+# Announcing SWR 2.0
+
+<Authors date="">
+</Authors>
+
+We are excited to introduce SWR v2.0, including features to improve the use experience of your application.
+
+## What’s New
+
+### Better React 18 Support
+
+SWR now uses `useSyncExternalStore` and `startTransition` internally to improve React 18 support, which ensures stronger UI consistency in concurrent rendering with React 18. The change is an internal implementation detail, so developers using SWR do not have to care about the change.
+
+### isLoading State
+
+`isLoading` is a new return value that indicates if there's an ongoing request and no "loaded data". Previously, you have to check `data` and `error` are `undefined`, but now you can use the `isLoading` value.
+
+You can improve the user experience by combining with `isLoading` and `isValidating` like this.
+
+```jsx
+function Stock() {
+  const { data, isLoading, isValidating } = useSWR(STOCK_API, fetcher, {
+    refreshInterval: 3000
+  });
+
+  // If it's still loading the initial data, there is nothing to display.
+  // We return a skeleton here.
+  if (isLoading) return <div className="skeleton" />;
+
+  // Otherwise, display the data and a spinner that indicates a background
+  // revalidation.
+  return (
+    <>
+      <div>AAPL ${data}</div>
```

Comment:
not sure what's AAPL, is it apple stock? wonder we can have less obscure working here to make the example more straightforward

Reply:
👍 I've picked up the example from https://github.com/vercel/swr/releases/tag/2.0.0-beta.1

Reply:
yeah but if other languages want to translate, less special abbr words will make it easy. Maybe just `<div>${data}</div>`